docs/docs/libraries/nodejs/getting_started.md

```diff
@@ -0,0 +1,28 @@
+# Getting Started with Node.js
+
+The [IOTA Wallet Node.js binding](https://www.npmjs.com/package/@iota/wallet) is published on [npmjs.com](https://www.npmjs.com/).
+
+:::info
+You can find a guide for exchanges and the most common use cases in the [Chrysalis documentation](https://chrysalis.docs.iota.org/guides/exchange_guide.html), which is based on `Wallet.rs` and `Node.js`. 
+:::
+
+## Security
+:::warning
+It is not recommended to store passwords on the host's environment variables, or in the source code in a production setup. 
+Please make sure you follow our [backup and security recommendations](https://chrysalis.docs.iota.org/guides/backup_security.html) for production use.
+:::
+## Installation
+
+Currently, the package isn't published,  so you'll need to link it to your project using  _npm_  or  _yarn_ . We also use  _dotenv_  for password management in the examples.
```

Comment:
It's published here https://www.npmjs.com/package/@iota/wallet

Reply:
Updated.